Jenkinsfile-Lane-Update

```diff
@@ -22,7 +27,9 @@ pipeline {
           # Make a PR
           git checkout -B lane-ezproxy-update
           git add config/ezproxy/lane_proxy_file.txt
-          git commit -m "Update Lane Library EZProxy config" && git push origin lane-ezproxy-update && hub pull-request -f -m "Update Lane Library EZProxy config"
+          git commit -m "Update Lane Library EZProxy config" &&
+          git push https://${GH_CREDS_USR}:${GH_CREDS_PSW}@github.com/sul-dlss/SearchWorks.git lane-ezproxy-update &&
```

Comment:
i though the sshagent let us push to e.g. git@github.com:sul-dlss/SearchWorks.git?

Reply:
maybe it does? I can try it.